Gems/Multiplayer/Code/Source/Pipeline/NetworkPrefabProcessor.cpp

```diff
@@ -158,6 +160,20 @@ namespace Multiplayer
             netEntity->InvalidateDependencies();
             netEntity->EvaluateDependencies();
 
+            auto* transformComponent = netEntity->FindComponent<AzFramework::TransformComponent>();
+            if (transformComponent)
+            {
+                AZ::EntityId parentId = transformComponent->GetParentId();
+                if (parentId.IsValid() && !prefabNetEntityIds.contains(parentId))
+                {
+                    // Clear parent ID for net entities parented to a non-net entity.
+                    // To be addressed by the spawnable aliases system.
```

Comment:
nit: May be worth clarifying either how this would be addressed ie this code block can be removed or will be updated. Also if the spawnable aliases has an RFC number it maybe useful to add it here.